qsharp.html.markdown

```diff
@@ -58,97 +60,102 @@ using (qs = Qubit[2]) {
 /////////////////////////////////////
 // 2. Classical data types and operators
 
-// Numbers in Q# can be stored in Int, BigInt or Double.
-let i = 1;            // This defines an Int variable i equal to 1
-let bi = 1L;          // This defines a BigInt variable bi equal to 1
-let d = 1.0;          // This defines a Double variable d equal to 1
+function ClassicalDataTypes() : Unit {
```

Comment:
Is there a way to not put everything in functions? It's annoying to permanently take up screen space for an indent, especially if you're reading the article on a phone that would be like 20% of your entire screen

Reply:
I was trying to make the whole file runnable. As are no global-scope statements, that meant wrapping things. Is it ok for learn x in y if the code snippet doesn't actually work? If so, we could just remove the function declarations.

Reply:
Hi @verhovsky  -- I did some digging and it does look like other samples try to preserve the "runnability" of the code snippet ([java example](https://github.com/adambard/learnxinyminutes-docs/blob/master/java.html.markdown)). Just wondering if you have any more thoughts on this.

Reply:
nope, sounds good to me